Gemfile

```diff
@@ -21,25 +21,23 @@ gem 'net-ldap'
 
 # Use puma as the webserver in development
 gem 'puma'
-gem 'rack', '1.6.4' # Remove in rails 5
```

Comment:
just want to make sure that the bug in that rack 2.0.1 doesnt have the same bug as 1.6.4, which was latest at the time of  [this commit](https://github.com/dmann/duke-data-service/commit/25d63bb8c431833611114f4f33e4d7915540af94) does not still have a bug. I wish I had better described the symptoms of the bug :(